torch/testing/_internal/common_device_type.py

```diff
@@ -1213,14 +1213,25 @@ def skipCUDAIfNoMagma(fn):
     return skipCUDAIf('no_magma', "no MAGMA library detected")(skipCUDANonDefaultStreamIf(True)(fn))
 
 def has_cusolver():
-    version = _get_torch_cuda_version()
+    cuda_version = _get_torch_cuda_version()
     # cuSolver is disabled on cuda < 10.1.243
-    return version >= (10, 2)
+    return cuda_version >= (10, 2)
+
+def has_hipsolver():
+    rocm_version = _get_torch_rocm_version()
+    # hipSOLVER is disabled on ROCM < 5.2
+    return rocm_version >= (5, 3)
```

Comment:
At few places in the code I see check for >=5.2 ROCm version. Checking will it be good to move all checks to ROCm >=5.3, then while upstreaming the patch it will be more clear, seeing some part enabled for 5.2 and some part for 5.3 could be confusing. Let me know your opinion.

Reply:
I think that's a reasonable suggestion. I assume rocm 5.3 is backwards compatible with 5.2 so make sense to bump the checks up for consistency.